src/main/java/com/knighttodo/knighttodo/rest/TodoBlockResource.java

```diff
@@ -48,10 +49,10 @@
     }
 
     @PostMapping("/block")
-    public ResponseEntity<CreateTodoBlockResponse> addTodoBlock(@RequestBody CreateTodoBlockRequest createRequest) {
+    public ResponseEntity<CreateTodoBlockResponse> addTodoBlock(
+        @Valid @NotEmpty @RequestBody CreateTodoBlockRequest createRequest) {
```

Comment:
You shouldn't use @NotEmpty here, it's better to use it inside the CreateTodoBlockReques on each field. And it's better to use @NotBlank annotation there.

Reply:
ok, updated it!